dataclients/kubernetes/kube.go

```diff
@@ -49,6 +49,7 @@ const (
 	ratelimitAnnotationKey        = "zalando.org/ratelimit"
 	skipperfilterAnnotationKey    = "zalando.org/skipper-filter"
 	skipperpredicateAnnotationKey = "zalando.org/skipper-predicate"
+	skipperRoutesAnnotationKey    = "zalando.org/skipper-routes"
```

Comment:
This should not be needed, but reuse the predicate key.  
We want to migrate also all routes magic, which are similar to predicates into predicates.

Reply:
But this is supposed to work differently than how the predicate annotation currently works. The predicate annotation just takes the predicates and adds them to the existing routes generated by the rules section of the ingress.

The routes annotation on the other hand is meant to create custom routes additional to the ones created from the ingress rules section.

These are two different behaviors so I believe it should be split into two different annotations.

Reply:
Thanks for your explanation. Then a new key makes sense.